Solutions/Corvus.Json.CodeGenerator/GenerateCommand.cs

```diff
@@ -21,6 +21,7 @@ public sealed class Settings : CommandSettings
     {
         [CommandOption("--rootNamespace")]
         [Description("The default root namespace for generated types.")]
+        [NotNull]
```

Comment:
So. That is what I understand NotNull to mean.

The issue is that the command line stuff seems to require that we have a get; init; property on that settings class, which means it has to be a nullable value.

However, the way I have defined the option means that it is required (and will throw in the magic of that command line framework if it is not set before we ever get to use those settings).

That attribute was an attempt to let users (and the compiler) know the situation.

I'm open to a better workaround. (I could just throw if it is null, for example.)

Reply:
I would suggest a comment explaining that the command line framework is on the hook for ensuring non-nullness here. (The command line framework seems to be a fast moving target, as they grapple with the balance between perf concerns such as trimmability, and API usability. So I think if we are making assumptions about what it will or won't enforce for us, those assumptions should be made explicit, so that we can review them when we take updates to the command line framework.)